xDiffCollectionTests/CollectionTestStatusTests.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  CollectionTestStatusTests.swift
+//  CollectionTestTests
+//
+//  Created by Esteban Garro on 2018-06-05.
+//  Copyright © 2018 aldo. All rights reserved.
+//
+
+import XCTest
+@testable import xDiffCollection
+
+fileprivate let object1 = CollectionTestObjectMock(value:"apple")
+fileprivate let object2 = CollectionTestObjectMock(value:"ansi")
+fileprivate let object3 = CollectionTestObjectMock(value:"bacon")
+fileprivate let object4 = CollectionTestObjectMock(value:"cocoon")
+
+class CollectionTestStatusTests: XCTestCase {
+    
+    
+    var myCollection : DiffCollection<CollectionTestObjectMock>?
```

Comment:
why do you need? optional here? if we sure that in setUp we will create a collection for test?